src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/OrganisationIdentifierValidatorImpl.java

```diff
@@ -14,7 +14,7 @@
 
 @Component
 @Slf4j
-public class OrganisationIdentifierIdentifierValidatorImpl implements OrganisationIdentifierValidator {
+public class OrganisationIdentifierValidatorImpl implements OrganisationIdentifierValidator {
 
```

Comment:
create validator package inside request and impl . move respective classes inside the package and apply same for test classes as-well

Reply:
This has been addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/450/commits/2e6a2f8b1a7050bb52d904bd20a2c5638869a566